src/schemas/json/rust-project.json

```
@@ -0,0 +1,126 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "crate": {
+      "type": "object",
+      "properties": {
+        "display_name": {
+          "type": "string",
+          "description": "Crate name used for display purposes, without affecting semantics."
```

```suggestion
          "description": "Crate name used for display purposes, without affecting semantics.",
          "type": "string"
```

Optional fix (available in multiple places): put description at first to encounter it earlier while reading from top to bottom and quicker understand what property is used for.